doc/rtd/reference/datasources/openstack.rst

```diff
@@ -36,7 +37,7 @@ The settings that may be configured are as follows:
 ``metadata_urls``
 -----------------
 
-This list of URLs will be searched for an OpenStack metadata service. The
+This list of URLs will be searched for an OpenStack IMDS. The
```

Comment:
There's a few `IMDS` references floating around in this file. We should either define it or change them all.